src/table/TableBodyWrapper.jsx

```diff
@@ -34,6 +34,13 @@ const TableBodyWrapper = ({ rootElement, tableData, constraints, selectionsAPI,
     colIdx: -1,
     isEnabled: !!selectionsAPI && !constraints.active,
   });
+  const handleMouseUp = (evt, cell) => {
```

Comment:
Good that you started on this, but it should not be in this PR since it is not part of the issue.